File: WebAdmin/Models/Feedback.cs
using System;

namespace WebAdmin.Models
{
    public class Feedback
    {
        public int FeedbackId { get; set; }
        public ushort Type { get; set; }
        public User FeedbackUser { get; set; }
        public string Description { get; set; }
        public DateTime FeedbackDate { get; set; }
        public ushort Stars { get; set; }
    }
}

File: src/WebAdmin/src/Models/Feedback.js
class Feedback {
    Id;
    Type;
    ClientEmail;
    Description;
    Date;
    Stars;

    constructor(id = -1, type = -1, clientEmail = '', description = '',
      date = '', stars = ''){
       this.Id = id;
       this.Type = type;
       this.ClientEmail = clientEmail;
       this.Description = description;
       this.Date = date;
       this.Stars = stars;
    }
}